test.js

```diff
@@ -48,4 +48,29 @@ describe('csslint-stylish', () => {
 
     assert(CSSLint.hasFormat('stylish'), 'csslint should be stylish')
   })
+
+  it('should be able to report from CSSLint without undefined line output', () => {
+    // assert(!CSSLint.hasFormat('stylish'), 'csslint should not be stylish')
+
+    CSSLint.addFormatter(reporter)
```

Comment:
Will check out that, I was not being able to run coverage successfully and because of that I needed to add a specific test to watch out and debug the output.:+1: 

I wanted a failing test case being pass when fixed as TDD implies...